modules/web_app/frontend/src/components/App/App.js

```diff
@@ -57,6 +58,7 @@ function App() {
               <Route exact path="/dup-report/:id" component={DupReport} />
               <Route exact path="/dup-compare" component={() => <DupCompare />} />
               <Route exact path="/dup-finder" component={() => <DupCompare />} />
+              <Route path='*' exact={true} component={ErrorPage} />
```

Comment:
Mình không cần exact cho 404 routing
```suggestion
              <Route path='*' component={ErrorPage} />
```